src/providers/dataProvider/resource-callbacks/ProjectLifeCycle.ts

```diff
@@ -1,31 +1,17 @@
-import { type AuditFunctionType, withCreatedByAt } from '../dataprovider-utils'
-import { AuditType } from '../../../utils/activity-types'
 import {
-  type ResourceCallbacks,
-  type CreateResult,
-  type UpdateResult
-} from 'ra-core'
+  type AuditFunctionType,
+  withCreatedByAt,
+  extendLifeCycle
+} from '../dataprovider-utils'
+import { type ResourceCallbacks, type CreateResult } from 'ra-core'
 import { R_PROJECTS } from '../../../constants'
 
-export default (audit: AuditFunctionType): ResourceCallbacks<any> => ({
```

Comment:
@Taha. Note: The only actual functionality in this module is this `createdAt` stuff.  Let's not change this PR, but in #95 I'm going to switch all types to use ResourceCreatedAt.  Then we can do `withCreatedAt` in the extendLifeCycle code.